packages/react-query-devtools/src/ReactQueryDevtools.tsx

```diff
@@ -42,6 +42,10 @@ export interface DevtoolsOptions {
    * Use this so you can attach the devtool's styles to specific element in the DOM.
    */
   shadowDOMTarget?: ShadowRoot
+  /**
+   * Set this to true to hide disabled queries from the devtools panel.
+   */
+  hideDisabled?: boolean
```

Comment:
can we rename this to `hideDisabledQueries` ? It’s kinda ambiguous with the current name.

Reply:
done!